README.md

```diff
@@ -139,7 +139,7 @@ Example using Redis:
 ### Consumer
 - `CLIENT SETNAME <group.id>:<name>`: sets the consumer's group & name
 - `CLIENT GETNAME`
-- `BLPOP topics:<topic> <timeoutMs>`: consumes the next message from the given topic
+- `BLPOP topics:<topic>|<JSON-encoded consumer config> <timeoutMs>`: consumes the next message from the given topic
```

Comment:
Just a thought: doesn't the following makes it easier to parse?

```
BLPOP topic:<topic> <timeoutMS>|<config>
```

...where the `|...` part is optional?

Reply:
As we discussed in person, let's keep redis semantics:

```
BLPOP topic:<topic>:<config> <timeoutMS>
```